src/Microsoft.OpenApi.Readers/V2/OpenApiHeaderDeserializer.cs

```diff
@@ -82,5 +150,27 @@ public static OpenApiHeader LoadHeader(ParseNode node)
 
             return header;
         }
+
+        private static void LoadStyle(OpenApiHeader h, string v)
+        {
+            switch (v)
+            {
+                case "csv":
+                    h.Style = ParameterStyle.Simple;
+                    return;
+                case "ssv":
+                    h.Style = ParameterStyle.SpaceDelimited;
+                    return;
+                case "pipes":
+                    h.Style = ParameterStyle.PipeDelimited;
+                    return;
+                case "tsv":
+                    throw new NotSupportedException();
+                case "multi":
+                    h.Style = ParameterStyle.Form;
+                    h.Explode = true;
+                    return;
```

Comment:
"multi" is now allowed in the header in V2 spec. Looks like it's only allowed in Param.